pulsar-broker/src/test/java/org/apache/pulsar/client/api/KeySharedSubscriptionTest.java

```diff
@@ -1630,4 +1630,63 @@ public void testContinueDispatchMessagesWhenMessageDelayed() throws Exception {
         log.info("Got {} other messages...", sum);
         Assert.assertEquals(sum, delayedMessages + messages);
     }
+
+    @Test
+    public void test()
```

Comment:
can we add a more meaningful test ?

Reply:
oops.. sorry, I just added this test to create an issue. let me fix tests and naming.